WebDriverAgentLib/Commands/FBSessionCommands.m

```diff
@@ -202,13 +200,35 @@ + (NSArray *)routes
           @"simulatorVersion" : [[UIDevice currentDevice] systemVersion],
           @"ip" : [XCUIDevice sharedDevice].fb_wifiIPAddress ?: [NSNull null],
           @"currentLocale": currentLocale,
-          @"timeZone": timeZone,
+          @"timeZone": [self getTimeZone],
         },
       @"build" : buildInfo.copy
     }
   );
 }
 
+/**
+ * @return The string of TimeZone. Returns TZ timezone id by default. Returns TimeZone name by Apple if TZ timezone id is not available.
+ */
++ (NSString *)getTimeZone
+{
```

Comment:
I would say it makes sense to extract time/locale info into a separate device info endpoint and not mix it with the status output